src/solidity-frontend/solidity_template.h

```diff
@@ -0,0 +1,386 @@
+/*
+  The template/library for Solidity built-in variables, function and data structure
+*/
+
+#include <nlohmann/json.hpp>
+#include <unordered_set>
+
+#ifndef SOLIDITY_TEMPLATE_H_
+#define SOLIDITY_TEMPLATE_H_
+
+namespace SolidityTemplate
+{
+/// header & typedef
+
+const std::string sol_header = R"(
+#include <stddef.h>
+#include <stdlib.h>
+#include <stdint.h>
+#include <string.h>
+)";
+
+/*
+  uint == uint256_t
```

Comment:
Do they contain the same number of bits?

  uint == uint256_t
  bytes == uint256_t
  bytes32 == uint256_t

Reply:
Yes, I believe so.